src/appState.ts

```diff
@@ -68,6 +68,7 @@ export const getDefaultAppState = (): Omit<
     width: window.innerWidth,
     height: window.innerHeight,
     isLibraryOpen: false,
+    tooManyElements: 1250,
```

Comment:
Let's add this as a constant to `src/constants.ts` instead of appState. AppState should be reserved for, well, app state.

Reply:
EDIT: except, we probably don't want a `tooManyElements` solution, see comments below. (But my comment above applies in general.)